.github/workflows/report_opencollective.yaml

```
@@ -0,0 +1,35 @@
+name: telegram github opencollective report
+on:
+  schedule:
+    - cron: "0 13 * * *"
```

```suggestion
    - cron: "0 13 26 * *"
```

Let's set this at the end of the month 👍